arcgis-ios-sdk-samples/Route & Directions/Find closest facility to multiple incidents (service)/README.md

```diff
@@ -2,7 +2,7 @@
 
 Find routes from several locations to the respective closest facility.
 
-![Screenshot of Find Closest Facility to Multiple Incidents Service sample](image1.png)
+![Screenshot of Find Closest Facility to Multiple Incidents Service sample](find-closest-facility-service.png)
```

Comment:
Oops. I didn't rename the image files for #838 and #842 . Should we rename the images to reflect the name of the example? @philium

Reply:
The only worry is for the metadata json file.

In the last sprint review meeting, Mike mentioned that the metadata files will be automatically updated based on the README file with a script. Hopefully these renames won't break anything on the website! I'll rename the images based on example names.